Fix Subclause name for cstddef in headers.cpp.fs

winspool opened the pull request:
The new name "Common definitions" for the entry of section 17.2 in Table headers.cpp.fs
is already used as name for the entry of section 17.2 in Table support.summary
and is also used as the name in section 17.2 (support.types)

There is still a difference for cstdlib (17.2.2: cstdlib.syn):
Table "support.summary" lists cstdlib for entry 17.2 and entry 17.5, but 
Table "headers.cpp.fs" lists cstdlib only for entry 17.5.

According to the Text in 17.2.2, cstdlib has 2 freestanding entries related to section 17.2
and 5 freestanding entries related to section 17.5.

cstdlib has references to a lot more sections, which are not listed in Table "support.summary",
but adding every reference from cstdlib to Table "support.summary" makes the Table very ugly.

--
Regards ... Detlef Riekenberg

jensmaurer reviewed (approved):
This seems to be an obviously-correct improvement.

@tkoeppe, when merging, fix the commit description. Referring to table numbers is unstable.

jensmaurer commented:
@winspool, if you want your full name to appear in the commit history, please adjust your github settings accordingly.

Feel free to force-push afterwards.

winspool commented:
I replaced the Table numbers with the Table names in the commit message
and I updated my Github Name
(as suggested by @jensmaurer)

Thanks for the Hints
-- 
Regards ... Detlef

winspool commented:
What can i do to get the fix commited?
@jensmaurer 
@tkoeppe 

-- 
Regards ... Detlef

tkoeppe commented:
Sorry for the delay, and thanks for the fix!
